kovri.md

```diff
@@ -9,12 +9,12 @@ Ring signatures obscure the sender.
 Stealth addresses prevent outside observers from knowing the destination of where Monero is sent to.
 And RingCT hides the amount of Monero that's transmitted.
 However, some personally identifiable information may be leaked at the network level when making a transaction.
-This is where Kovri comes in to fill this possible gap.
+This gap of privacy is closed by a technology called Kovri.
```

Comment:
This sentence sounds so weird to me.

Reply:
How about something like "This privacy leak is fixed by a technology called Kovri".